modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -22,20 +25,26 @@
 #   specific machines.
 #   Default: true
 #
+# [*version*]
+#   The version of elasticsearch to install.  This must specify an exact
+#   version (eg 1.4.2)
+#
 class govuk_elasticsearch (
   $version,
   $cluster_hosts = ['localhost'],
   $cluster_name = 'elasticsearch',
   $heap_size = '512m',
   $number_of_shards = '3',
   $number_of_replicas = '1',
-  $minimum_master_nodes = '1',
+  $minimum_master_nodes = '2',
```

Comment:
I don't see the harm of adding the parameter if it has use in an environment such as vagrant. I'm not a big fan of hardcoding values when most of the other values have already been set as parameters. That is, as long as the change does not affect the current default behviour.